docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -0,0 +1,257 @@
+    ```
+
+    After you run this command, the Terminal will look like it's frozen. This is normal – it's running a program in the background.
+
+    b. Open your web browser (like Safari or Chrome).
+
+    c. In the address bar at the top of the browser window, type exactly this:
+
+    ```
+    http://localhost:3000
+    ```
+
+    Then press `Enter`.
+
+    d. The Odigos control panel should now open in your browser. It might take a few seconds to load.
+
+    #### 5. Set up Odigos
+
+    a. In the Odigos control panel, you'll see a page titled "Select Sources". Sources are the things Odigos can monitor.
+
+    b. Click the checkbox next to each Source you want to monitor. If you're not sure, you can select all of them.
+
+    c. Click the blue "Next" button at the bottom right of the page.
+
+    d. You'll now see a page titled "Add Destinations". Destinations are places where Odigos can send its information.
+
+    e. Click on one of the destination options (for example, "Jaeger").
+
+    f. Fill in any required information. If you're not sure what to enter, consult the Odigos documentation or ask for help.
+
+    g. Click the blue "Create Destination" button.
+
+    h. You should now see an overview page showing your selected Sources and Destinations.
+
+    Congratulations! You've now installed and set up Odigos on your Mac.
+
+  </Tab>
+  
+  <Tab title="Linux">
+
+  **Docker**: Creates a space on your computer where Odigos can run.
+  - Open a terminal window.
+  - Copy and paste these commands, pressing `Enter` after each:
+    ```
+    sudo apt-get update
+    sudo apt-get install docker.io
+    sudo systemctl start docker
+    sudo systemctl enable docker
+    ```
+  
+  **KinD**: Helps set up a test environment for Odigos.
+  - In the terminal, copy and paste these commands, pressing `Enter` after each:
+    ```
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
+    ```
+  </Tab>
+
+
+  <Tab title="Windows">
+  **Docker Desktop**: Creates a space on your computer where Odigos can run.
+  - Download Docker Desktop from the [official website](https://www.docker.com/products/docker-desktop).
+  - Run the installer and follow the on-screen instructions.
+  - Once installed, start Docker Desktop from the Start menu.
+
+  **Windows Subsystem for Linux (WSL)**: Allows you to run Linux tools on Windows.
+  - Open PowerShell as Administrator (right-click on PowerShell in the Start menu and select "Run as administrator").
+  - Type this command and press `Enter`:
+    ```
+    wsl --install
+    ```
+  - Restart your computer when prompted.
+
+  **KinD**: Helps set up a test environment for Odigos.
+  - Open PowerShell.
+  - Type these commands and press `Enter` after each:
+    ```
+    curl.exe -Lo kind-windows-amd64.exe https://kind.sigs.k8s.io/dl/v0.11.1/kind-windows-amd64
+    move kind-windows-amd64.exe C:\Windows\system32\kind.exe
+    ```
+  </Tab>
```

Comment:
I think we should either remove these or make sure they are consistent with the MacOS section